.github/templates/metagraph_l0/genesis/action.yml

```diff
@@ -13,6 +13,11 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Update versions.sbt to github-action-test-version
+      shell: bash
+      run: |
+        rm versions.sbt
+        echo "ThisBuild / version := \"github-action-test-version\"" >> versions.sbt
```

Comment:
couldn't you use single quotes instead if escaping the double quote?

Reply:
I'll take a look, probably. I've just opened the PR to test the action

Reply:
Also, just curious: why do an `rm` and an append (`>>`) instead of just `>` ?